SCF/SCF-session1/Assignment-1/ShopingCart.java

```diff
@@ -0,0 +1,213 @@
+import java.util.*;
+/**represent the Shoping Cart
+ * @author Ramdev
+ * this class have an another class "Item",
+ */
+public class ShopingCart 
+{
+	public static void main(String[] args) 
+	{
+		Scanner input = new Scanner(System.in);
+		ShopingCart cart = new ShopingCart();
+		int choice = 0,ID = 0,quantity;
+		boolean flag = true;
+		while( flag )
+		{
+			System.out.println("Enter the operation your want to perform");
+			System.out.println("1. Add item");
+			System.out.println("2. Remove item");
+			System.out.println("3. Update Quantity");
+			System.out.println("4. Display the Items");
+			System.out.println("5. Generate the Bill");
+			System.out.println("6. if you are done !");
+			choice = input.nextInt();
+			switch( choice )
+			{
+			   case 1:  ShopingCart.showItems();
+			   			System.out.println("Select the ID of Item your want to add");
+			   			ID = input.nextInt();
+			   			System.out.println("Enter the Quantity of the Item "+itemList[ID-1]);
+			   			quantity = input.nextInt();
+			   			cart.addItem(itemList[ID-1], itemPrices[ID-1], quantity);
+			   			break;
+			   case 2:  cart.displayItems();
+						System.out.println("Select the ID of Item you want to remove ");
+						ID = input.nextInt();
+						cart.removeItem(ID);
+				   		break;
+			   case 3:  cart.displayItems();
+	   					System.out.println("Select the ID of Item your whoose quantity you want to update");
+	   					ID = input.nextInt();
+	   					System.out.println("Enter the update Quantity of the Item "+cart.items[ID-1].name);
+	   					quantity = input.nextInt();
+	   					if(quantity == 0)
+	   						cart.removeItem(ID);
+	   					else
+	   					    cart.updateQuantity(cart.items[ID-1].name,quantity);
+	   					break;
+			   case 4: 	cart.displayItems();
+			   			break;
+			   case 5: 	System.out.println("your total bill is " + cart.getBill());
+			   			break;
+			   case 6: 	flag = false;
+			   			break;
+			   default: System.out.println("please Input right choice ");		   			
+			}
+		}
+	}
+	private int noOfItems;
```

Comment:
variable name : noOfItems -> cartTotalQuantity (sum of quantity of all items in cart)